test/integration/crcsuite/crcsuite.go

```diff
@@ -90,7 +90,7 @@ func FeatureContext(s *godog.Suite) {
 			os.Exit(1)
 		}
 
-		if bundleLocation == "embedded" {
+		if bundleLocation == "" {
```

Comment:
Nothing in the codebase is using --bundle-location="embedded"? To see a change in this commit removing this, or changing a default value, ...

Reply:
I think it is only used by the internal jenkins job where QE test the embedded release, they pass the `--bundle-location` as `embedded` which I think not required.

Reply:
Yes, I think it is only used in the Makefile for the purposes of the CI or local testing with embedded bundle. This is a solution from the time when there was no way to distinguish the `crc` binary cases (embedded vs not) except by size :). As you say, it might not be needed now.

Reply:
@jsliacan Also moving forward we are not going to use `embedded` (no bundle embed for binary) because of signing issue so I think better to remove this completely.